pkg/cmd/node/node_diagnostics_status.go

```diff
@@ -16,6 +16,11 @@ func newCmdNodeDiagnosticsStatus(ctx api.Context) *cobra.Command {
 		Short: "Print diagnostics job status",
 		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			err := ctx.Deprecated("This command is deprecated since DC/OS 1.14, please use 'dcos diagnostics' instead.")
```

Comment:
This should be `dcos diagnostics status`, right?

Reply:
That doesn't exist. :-D So we just defer the user to the next higher level of the cmd hierarchy.

Reply:
We could make it or we could point them to `diagnostics list` as well.